common/util.go

```diff
@@ -80,13 +80,13 @@ func ServerResourceForGroupVersionKind(disco discovery.DiscoveryInterface, gvk s
 }
 
 // SendSuccessResponse sends http success response
-func SendSuccessResponse(writer *http.ResponseWriter) {
-	(*writer).WriteHeader(http.StatusOK)
-	(*writer).Write([]byte(SuccessResponse))
+func SendSuccessResponse(writer http.ResponseWriter) {
+	writer.WriteHeader(http.StatusOK)
+	writer.Write([]byte(SuccessResponse))
 }
 
 // SendErrorResponse sends http error response
-func SendErrorResponse(writer *http.ResponseWriter) {
-	(*writer).WriteHeader(http.StatusOK)
-	(*writer).Write([]byte(SuccessResponse))
+func SendErrorResponse(writer http.ResponseWriter) {
```

Comment:
you should still be able to pass a pointer here.

Reply:
it is a pointer.
 https://stackoverflow.com/questions/13255907/in-go-http-handlers-why-is-the-responsewriter-a-value-but-the-request-a-pointer